frontend/i18next.scanner/fr/translation.json

```diff
@@ -630,5 +630,14 @@
   "You'll <1>receive a daily e-mail</1> summarizing everything you haven't already read.<3>There will be a single email for all spaces configured as \"Daily\".</3>": "Vous <1>recevrez un e-mail par jour</1> résumant tout ce que vous n'avez pas déjà lu.<3>Il n'y aura qu'un seul e-mail pour tous les espaces configurés en \"Quotidien\".</3>",
   "You'll <1>receive a weekly e-mail</1> summarizing everything you haven't already read.<3>There will be a single email for all spaces configured as \"Weekly\".</3>": "Vous <1>recevrez un e-mail par semaine</1> résumant tout ce que vous n'avez pas déjà lu.<3>Il n'y aura qu'un seul e-mail pour tous les espaces configurés en \"Hebdomadaire\".</3>",
   "You'll receive no e-mail notifications, even if you are mentioned.<1>You'll need to rely on recent activities and the notification wall to stay up to date.</1>": "Vous ne recevrez aucun e-mail de notification, même si vous êtes mentionné.<1>Vous devrez utiliser les activités récentes et le mur de notification pour rester à jour.</1>",
-  "logbook_search": "Journal"
+  "logbook_search": "Journal",
+  "Error while changing web notification subscription": "Une erreur est survenue lors du changement de l'abonnement aux notifications web.",
+  "Web notifications": "Notifications web",
+  "Enabled": "Activées",
+  "Disabled": "Désactivées",
+  "Web notifications:": "Notifications web :",
+  "Enable/Disable web notifications for this workspace.": "Activer / Désactiver les notifications web pour cet espace.",
+  "When disabled, notifications from this workspace will be hidden from your notification wall.": "Lorsque désactivées, les notifications de cet espace seront cachées de votre mur de notifications.",
```

Comment:
I'd say "Si désacrivées, ..." instead of "Lorsque désacrivées, ..."

Reply:
`Une erreur est survenue lors du changement de l'abonnement aux notifications web.` -> Une erreur est intervenue durant la modification de l'abonnement aux notifications web.`

`Lorsque désactivées, les notifications de cet espace seront cachées de votre mur de notifications.` - > `Si les notifications web sont désactivées, elles n'apparaîtront plus sur votre mur de notifications.`

Reply:
@Skylsmoi @Millefeuille42

Reply:
@lebouquetin 
Ok but I'd say 'survenue' instead of 'intervenue' since 'survenue' is what we have everywhere else in tracim

Reply:
fixed

Reply:
ok